tools/cli/go-whisk-cli/main.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
     // Rest of CLI uses the Properties struct, so set the build time there
     commands.Properties.CLIVersion = CLI_BUILD_TIME
+    os.Setenv("CLI_Version", CLI_BUILD_TIME)
```

Comment:
maybe this will work.
- add AgentHeader property to the whisk.Config struct
- in commands.go (line 54 or so), set the AgentHeader value in the client configuration init